### DISCUSSION TITLE
Where are the social features in 1.0.0?

### DISCUSSION THREAD

**Discussion by rzelnik** [beakerbrowser/beaker#1859] [Q&A]
If the social profile and social apps were removed from Beaker, will they be somehow available as standard web apps or as code examples or tutorials? Does it mean the end of general social profile?

**Comment by danrobi11** [comment]
Ya. Wheres the new Twitter-like ? Im running Beaker.Browser-1.0.0 and I've lost all my Hyperdrives for ?

**Reply by rzelnik** [reply]
Hyperdrives should be available, check beaker://library/drives

**Reply by danrobi11** [reply]
@rzelnik Yep my bad. Didnt see them mix between my contacts. Thanks!

**Comment by pfrazee** [comment]
Yeah I [tweeted a bit](https://twitter.com/pfrazee/status/1334230184220876800?s=20) about this ([more](https://twitter.com/pfrazee/status/1334333510895742978?s=20)). I pretty much cover the why there.

No data should be lost, like @rzelnik says. Right now, I think we should all bother @zootella about his dlog project, which will be a more interesting version of what we were doing.

[Some of the code is preserved here](https://github.com/beakerbrowser/social). Feel free to use it. [This template](https://beaker.dev/docs/templates/microblog-feed/) is also a good starting point.

**Reply by danrobi11** [reply]
Sad to hear , really is. I've seen a screenshot of what I thought was almost ready. Dang it!

**Reply by pfrazee** [reply]
Yeah I hate to disappoint. The code is still archived and sitting for the right moment, but it just wasn't quite ready.

If there are any teams that want to pick up the pieces of that code, I'd be more than happy to coach them through picking it up and deploying it in their own projects.

**Comment by alizehctl** [comment]
Does the deprecation of the social layer in Beaker affect the functioning of `blahbity-blog` and its sibling apps? 

Like, could we just keep on using and making the same apps, even though there isn't a "profile drive" in Beaker anymore? Or does the API they plug into no longer exist?

**Reply by alizehctl** [reply]
> Or does the API they plug into no longer exist?

I guess the new page for the [beaker.contacts](https://docs.beakerbrowser.com/apis/beaker.contacts) API answers that. So my takeaway is the old apps will continue to work, but making new apps is dubious since the `contacts` API is deprecated.

**Reply by pfrazee** [reply]
Blahbity should keep working, and we deprecated the contacts in favor of using drive tags, which we hope is more generic and no less capable.
